src/module/rules/rule-element/crafting/entry.ts

```diff
@@ -47,8 +48,12 @@ class CraftingEntryRuleElement extends RuleElementPF2e {
             requiredTraits: this.data.requiredTraits,
             maxItemLevel: this.data.maxItemLevel,
             maxSlots: this.data.maxSlots,
+            batchSize: this.data.batchSize,
+            fieldDiscovery: this.data.fieldDiscovery,
+            fieldDiscoveryBatchSize: this.data.fieldDiscoveryBatchSize,
+            fieldDiscoveryRequiresMundane: this.data.fieldDiscoveryRequiresMundane,
+            maxFieldDiscoveryItemLevel: this.data.maxFieldDiscoveryItemLevel,
```

Comment:
This is way too hard-coded for alchemy

Reply:
I am not sure if you wanted all of it gone, or just the mundane and maxItemlevel ones.

The others seemed to have just been missed, as they were properties on the entry before, and could make sense to set on add